### PR TITLE
Update CI to quick check benchmarks

### PR DESCRIPTION
**Pull request by Chralt98**
Fixes partially https://github.com/zeitgeistpm/zeitgeist/issues/822

### CODE REVIEW

**Comment by maltekliemann** [comment]
We can see that the script works, since it shows us that the `pallet-collective` benchmarks fail. Let's merge the fix for that, and then we can do the review here.

**Review by sea212 — APPROVED** [review]
Nice, this is helpful.
What's left to close https://github.com/zeitgeistpm/zeitgeist/issues/822 ? In case it's only the bugfix within `pallet_collective`, let's close it with the merge of this PR.

**Comment by Chralt98** [comment]
As soon as this PR is merged, #822 can be closed, as you pointed out. 👍 I will wait for the second review and your release in Discord.

**Comment by sea212** [comment]
Please respect point 2. at https://github.com/zeitgeistpm/zeitgeist/pull/841#issue-1416377613 before merging this.

File: scripts/benchmarks/run_benchmarks.sh
Comment (sea212) [inline scripts/benchmarks/run_benchmarks.sh:13]: I think this part logically belongs into *scripts/benchmarks/configuration.sh*.

File: scripts/benchmarks/quick_check.sh
Comment (sea212) [reply scripts/benchmarks/quick_check.sh:24]: If we use `dev` we'd also have to ensure that it uses `target/debug` instead of `target/dev` to executed the Zeitgeist binary.

Comment (Chralt98) [reply scripts/benchmarks/quick_check.sh:24]: c96054002660b666b62e4bb7c3f5f02264664b4c

File: scripts/benchmarks/run_benchmarks.sh
Comment (Chralt98) [reply scripts/benchmarks/run_benchmarks.sh]: It does indeed! c96054002660b666b62e4bb7c3f5f02264664b4c

**Review by sea212 — APPROVED** [review]
Great! Finally we check that benchmarks run through properly :tada:

**Review by maltekliemann — APPROVED** [review]
Looks good! :thumbsup:

**Pull request merged**
